kiwi/builder/template/container_import.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2024 SUSE Software Solutions Germany GmbH.  All rights reserved.
+#
+# This file is part of kiwi.
+#
+# kiwi is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# kiwi is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with kiwi.  If not, see <http://www.gnu.org/licenses/>
+#
+import os
+from string import Template
+from textwrap import dedent
+
+
+class BuilderTemplateSystemdUnit:
+    """
+    **systemd unit file templates**
+    """
+    def get_container_import_template(self) -> Template:
+        template_data = dedent('''
+            # kiwi generated unit file
+            [Unit]
+            Description=Import Local Container: ${container_name}
+            ConditionPathExists=${container_file}
+
+            [Service]
+            Type=oneshot
+            ExecStart=${load_command}
+            ExecStartPost=/bin/rm ${container_file}
+
+            [Install]
+            WantedBy=multi-user.target
+        ''').strip() + os.linesep
+        return Template(template_data)
```

Comment:
We probably want to have dependency relationships somehow expressed in the XML to translate to the units, yes.

Reply:
good point, I'll take a closer look once I've recovered from the *f* cold

Reply:
Also, if `docker` is set the unit file should have a `After: docker.service` entry since dockerd needs to be running.

Reply:
I added a commit that implements a solution which I believe is acceptable. kiwi will now produce only one service file called `kiwi_containers.service` It makes use of the one-shot service types which also allows for multiple execution of programs. Writing an extra script is not needed in this case. The implementation also takes care for eventual backend service requirements. So far we only have `docker.service` to run beforehand. If we combine docker, podman, containerd, etc backends in the future the service requirement for all of them has to be met, which I believe is also a wanted behavior. Also if one execution fails the complete service run is marked failed, that's also what we want because a mixed state of success and error is not good.